### PR TITLE
Vanillahill/disable concentrator

### PR DESCRIPTION
**Pull request by vanillaHill**
Closes https://github.com/FortressFinance/issues/issues/146

### CODE REVIEW

**Comment by vanillaHill** [comment]
Closes https://github.com/FortressFinance/issues/issues/145

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -39,6 +41,23 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
     state.setConnectModal,
   ])
 
+  let concentratorsItem = (
+    <Menu.Item as={DropdownMenuItemLink} href={appLink("/yield/concentrators")}>
+      Concentrators
+    </Menu.Item>
+  )
+
+  if (DISABLE_CONCENTRATORS) {
```
Comment (aquaticone) [inline src/components/Layout.tsx:50]: I'd probably just do it inline with a ternary but nbd.

File: src/components/Layout.tsx [hunk 2]
```diff
@@ -39,6 +41,23 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
...
+    concentratorsItem = (
+      <Menu.Item as={DropdownMenuItemLink} href={appLink("#")} disabled>
```
Comment (aquaticone) [inline src/components/Layout.tsx:52]: You can just do `href="#"`. I think you should preventDefault when the user clicks this too though.

File: src/lib/helpers/toBool.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+type Falsey = undefined | null
```
Comment (aquaticone) [inline src/lib/helpers/toBool.ts:1]: seems kinda unnecessary to declare this as it's own type

File: src/components/Layout.tsx (hunk 2 above)
Comment (aquaticone) [reply src/components/Layout.tsx:52]: Also, vertical alignment is just *slightly* off. If you put `flex items-center` on this it will be perfect.

**Review by aquaticone — APPROVED** [review]
Looks good sir

**Pull request merged**